docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
+      - my_media:/files/media
+    command:
+      sh -c "python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
```

Comment:
+ Implement wait_for_db [management command](https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/), which waits for the database to be available. So your services won't throw any errors during the docker-compose up command;

(Task from README.md)